spec/support/user.rb

```diff
@@ -5,9 +5,13 @@ class RawUser < ActiveRecord::Base
 end
 
 class User < ActiveRecord::Base
+  DEFAULT_DATE = ::Date.new(2019, 7, 17)
+
   store_accessor :jparams, :version, active: :boolean, salary: :integer
   store_attribute :jparams, :birthday, :date
-
+  store_attribute :jparams, :static_date, :date, default: DEFAULT_DATE
+  store_attribute :jparams, :dynamic_date, :date, default: -> { DEFAULT_DATE }
```

Comment:
Let's use different values for dynamic and static defaults. And it would be great to make dynamic really dynamic) 

It's not clear why `jamie.static_date` and `jamie.dynamic_date` are the same in tests.